main.py

```diff
@@ -104,6 +107,9 @@ def remove_mw(self, mw_id):
         except KeyError:
             return jsonify({'response': f'Maintenance with id {mw_id} not '
                                         f'found'}), 404
+        if maintenance.status == Status.RUNNING:
+            return jsonify({'response': f'Deleting a running maintenance is '
+                                        f'not allowed'}), 400
```

Comment:
I think this `f'` (f-string) can be safely removed.

Reply:
Done. You can add the modifications as code suggestions so that it's easier to just accept+commit the changes, @cmagnobarbosa .